### PR TITLE
[issue-104] Fixed incorrect hadoop connector artifact reference

### LINKED ISSUE #104: Fix incorrect hadoop connector artifact reference
The fix from https://github.com/pravega/pravega-samples/pull/77 introduced few changes to the build configurations that appears incorrect.
- `gradle.properties` is added to `hadoop-examples` sub-module. Instead, we could use the project's root `gradle.properties`  (move all the configurations to root gradle properties and delete this file)
https://github.com/pravega/pravega-samples/blob/master/hadoop-examples/gradle.properties

- Rename the variable from `connectorVersion` to `hadoopConnectorVersion` as it conflicts with the flink connector version variable name
https://github.com/pravega/pravega-samples/blob/master/hadoop-examples/gradle.properties#L7

- Update the hadoop connector version (snapshot) to the latest version published in jcenter repo
```
#hadoop connector
hadoopVersion=2.8.1
scalaVersion=2.11.8
sparkVersion=2.2.0
hadoopConnectorVersion=0.3.0-17.a171316-SNAPSHOT
```

- Fix the artifact ID of the hadoop connector i.e., from `hadoop-connectors` to `pravega-connectors-hadoop`
https://github.com/pravega/pravega-samples/blob/master/hadoop-examples/build.gradle#L38
https://github.com/pravega/pravega-samples/blob/master/hadoop-examples/build.gradle#L47

The fix should be patched on `develop` branch.

### PR DESCRIPTION
**Pull request by vijikarthi**
Signed-off-by: Vijay Srinivasaraghavan <vijayaraghavan.srinivasaraghavan@emc.com>

Fixes #104 : 
* moved gradle properties configurations from `hadoop-examples` to root
* updated hadoop connector snapshot version

### CODE REVIEW

File: hadoop-examples/build.gradle [hunk 1]
```diff
@@ -44,7 +43,7 @@ dependencies {
 shadowJar {
     version = version
     dependencies {
-        include dependency("io.pravega:hadoop-connectors")
+        include dependency("io.pravega:pravega-connectors-hadoop")
```
Comment (RaulGracia) [inline hadoop-examples/build.gradle]: As the example uses classes from `pravega-client`, to make it work in my setting I have to add:
```
include dependency("io.pravega:pravega-client")
include dependency("io.pravega:pravega-common")
```

File: hadoop-examples/build.gradle [hunk 2]
```diff
@@ -35,7 +34,7 @@ repositories {
 }
 
 dependencies {
-    compile     "io.pravega:hadoop-connectors:${connectorVersion}"
+    compile     "io.pravega:pravega-connectors-hadoop:${hadoopConnectorVersion}"
```
Comment (RaulGracia) [inline hadoop-examples/build.gradle]: After trying the changes from this PR locally, the example fails throwing exceptions like: 
```
java.lang.Exception: io.pravega.client.segment.impl.NoSuchSegmentException: Segment does not exist:myScope/myStream/2
```
I found similar exceptions when testing examples including versions of `pravega-client` older than the major changes merged in the last days (e.g., rolling transactions). Thus, to make this example work out-of-the-box, maybe we will need to configure a newer version of the hadoop connector build with libraries containing the recent major changes in pravega.

Comment (vijikarthi) [reply hadoop-examples/build.gradle]: I have created an issue for this https://github.com/pravega/hadoop-connectors/issues/36 yesterday and @yangb8 is working on it.

File: hadoop-examples/build.gradle (hunk 1 above)
Comment (vijikarthi) [reply hadoop-examples/build.gradle]: Are you saying the hadoop connectors does not include shaded Pravega client libraries?

Comment (RaulGracia) [reply hadoop-examples/build.gradle]: This was a problem of my setting, so forget it. I tested the code of the PR and it works.

**Comment by vijikarthi** [comment]
Can we get this merged?

**Pull request merged**